### PR TITLE
DataGrid: Add more auto-generator attributes

### PR DESCRIPTION
**Pull request by David-Moreira**
- IgnoreFieldAttribute
- OrderAttribute
- NumericAttribute
- SelectAttribute
- DateAttribute

### CODE REVIEW

File: Source/Blazorise/Attributes/IgnoreFieldAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Blazorise;
+
+/// <summary>
+/// Ignores the field from being included.
```
Comment (stsrki) [inline Source/Blazorise/Attributes/IgnoreFieldAttribute.cs:10]: Should better define in what the field is not included.

File: Source/Blazorise/Attributes/NumericAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise;
+
+/// <summary>
+/// Configures a numeric field.
```
Comment (stsrki) [inline Source/Blazorise/Attributes/NumericAttribute.cs:7]: More descriptive comment.

eg. `Represents an attribute that can be applied to numeric properties or fields to specify additional metadata.`

File: Source/Blazorise/Attributes/OrderAttribute.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Blazorise;
+
+/// <summary>
+/// Configures the field order.
```
Comment (stsrki) [inline Source/Blazorise/Attributes/OrderAttribute.cs:6]: Also more descriptions.

PS. Chat GPT works well for this.

**Review by stsrki — COMMENTED** [review]
Other than the comments, looks good to me.

**Comment by David-Moreira** [comment]
Added also Select and Date Attributes, I think this attributes are fine for now as they complement the possible DataGrid columns.

File: Source/Blazorise/Attributes/SelectAttribute.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    /// The name of the function that will be used to get the data source.
+    /// By default it's set to "GetSelectData".
+    /// </summary>
+    public string GetDataFunction { get; set; } = "GetSelectData";
+
+    /// <summary>
+    /// Used to get the display field from the supplied data source.
+    /// Defaults to "Description".
+    /// </summary>
+    public string TextField { get; set; } = "Description";
```
Comment (stsrki) [inline Source/Blazorise/Attributes/SelectAttribute.cs:21]: Should we even have default names for the fields? The combination of `Description` and `Id` seems very rare in the real world.

Maybe name them differently?

- `Value`
- `Text`

Comment (David-Moreira) [reply Source/Blazorise/Attributes/SelectAttribute.cs:21]: Well rare for you...? :D I'm thinking in the perspective of someone binding a DTO.
So at least, I have worked with Id commonly. So wanting the **Id** as the **Value** which corresponds to the database entity. That seems like a common case? Even if its a DTO the dto generally also points to the Id. 

As for the **Text** I think it's generally Description or Text,..?

Comment (David-Moreira) [reply Source/Blazorise/Attributes/SelectAttribute.cs:21]: I also don't mind not assuming any defaults.
The idea was that if someone were to have the same type of binding, he would have to define less metadata in the attribute.

Comment (David-Moreira) [reply Source/Blazorise/Attributes/SelectAttribute.cs:21]: Here's an example on our own code base, for support forum, that if we used a `SelectList` the `ValueField` would be the `Id`
image

Comment (David-Moreira) [reply Source/Blazorise/Attributes/SelectAttribute.cs]: Done.

**Comment by stsrki** [comment]
Can you add release notes? I have already create an empty 1.6 page and merged it here.

**Review by stsrki — APPROVED** [review]

**Pull request merged**